plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHive3OnDataLake.java

```diff
@@ -80,22 +80,22 @@ public class TestHive3OnDataLake
     private static final DataSize HIVE_S3_STREAMING_PART_SIZE = DataSize.of(5, MEGABYTE);
 
     private String bucketName;
-    private HiveMinioDataLake hiveMinioDataLake;
+    private Hive3MinioDataLake hive3MinioDataLake;
     private HiveMetastore metastoreClient;
 
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.bucketName = "test-hive-insert-overwrite-" + randomNameSuffix();
-        this.hiveMinioDataLake = closeAfterClass(
```

Comment:
I think we can leave the private object name as-is hiveMinioDataLake (here and others). Reduces the diff, and also it looks like you are naming it all back when this is converted to an abstract class [later](https://github.com/trinodb/trino/pull/24418/commits/661fcec1974cfc6bba172b01ac83c87dce95eb2b#diff-c6ac76ce5a102f34fe5cdca7794cff7404bf8ed13740ae7232c25ad6205a1245).

Reply:
I personally like to keep the name as that of the class but I'll revert as you suggested. ya I renamed it later for one of the class as in that commit `Hive3MinioDataLake` becomes `HiveMinioDataLake`